CHANGELOG.md

```diff
@@ -1,13 +1,32 @@
 # Changelog
 
+## [0.4.0] - 2023-11-02
+
+<font size='7'>:rotating_light: ***PLEASE READ BEFORE UPGRADING*** :rotating_light:</font>
+
+ This version contains changes in the `ClusterSPIFFEID` CRD, `ClusterFederatedTrustDomain` CRD and `ClusterStaticEntry` CRD. Before upgrading you __MUST__ do the following, in order:
+
+- Update those CRDs into your cluster (see [here](./config/crd/bases/spire.spiffe.io_clusterspiffeids.yaml), [here](./config/crd/bases/spire.spiffe.io_clusterfederatedtrustdomains.yaml) and [here](.config/crd/bases/spire.spiffe.io_clusterstaticentries.yaml)).
+- Update the `manager-role` ClusterRole, which includes additional permissions for `endpoints` CRD (see [here](./config/rbac/role.yaml))
+
+### Security
+
+- Updated to google.golang.org/grpc v1.59.0 to address CVE-2023-44487
+
+### Added
+
+- ClusterSPIFFEID CRD can configure to Autopopulate DNS (#122)
+- Support to run multiple spire clusters within a k8s cluster (#230)
+- Status subresource definitions (#223)
```

Comment:
This should go under "Fixed" and then:

`missing status subresource definitions`